frontend/src/GetDatasetJulia.js

```diff
@@ -44,8 +44,12 @@ export default function GetDatasetJulia({dataset}) {
 # Dataset and column metadata are under dataset_metadata[x] and 
 # column_metadata[x] respectively.
 
+using Pkg
+
+Pkg.add(["HTTP", "JSON", "DataFrames"])
```

Comment:
Is it standard Julia practice to install packages within scripts? If so, good addition, thanks.

Reply:
I wouldn't say it's standard practice, but it is somewhat common for examples. In this case, it's helpful for new users to avoid package installation errors.